tasks/utils.py

```diff
@@ -61,7 +61,8 @@ def get_build_flags(ctx, static=False, use_embedded_libs=False):
     ldflags += "-X {}/pkg/serializer.AgentPayloadVersion={} ".format(REPO_PATH, payload_v)
     env = {
         "PKG_CONFIG_PATH": pkg_config_path(use_embedded_libs),
-        "CGO_CFLAGS_ALLOW": "-static-libgcc"  # whitelist additional flags, here a flag used for net-snmp
+        "CGO_CFLAGS_ALLOW": "-static-libgcc",  # whitelist additional flags, here a flag used for net-snmp
+        "CGO_LDFLAGS_ALLOW": "-Wl,--allow-multiple-definition"
```

Comment:
Since we don't know why this breaks on windows and not linux just yet, could we do this on windows only:
```python
if platform.system().lower() == "windows":
    env['CGO_LDFLAGS_ALLOW'] = '-Wl,--allow-multiple-definition'
```